playwright-e2e/tests/brain/brain-regression-test.spec.ts

```diff
@@ -0,0 +1,386 @@
+    await page.getByRole('img', { name: 'Keith Ligon photo' }).click();
+    await page.getByRole('img', { name: 'Roel Verhaak photo' }).click();
+    await page.getByRole('img', { name: 'W. K. Alfred Yung photo' }).click();
+    await page.getByRole('img', { name: 'Patrick Y. Wen photo' }).click();
+    await page.getByText('Brain TumorProject FAQs About Us Join Mailing List Data Log In Count Me In Menu').click();
+    await page.getByRole('banner').getByRole('listitem').filter({ hasText: 'Data' }).click();
+    await page.getByRole('heading', { name: 'We’re in the process of collecting data from participants across the United States and Canada.' }).click();
+    await page.getByRole('heading', { name: 'Snapshots' }).click();
+    await page.getByRole('img', { name: 'Brain tumor diagnosis type diagram' }).click();
+    await page.getByRole('button').filter({ hasText: 'clear' }).click();
+    await page.getByRole('img', { name: 'Age at initial brain tumor diagnosis diagram' }).click();
+    await page.getByRole('button').filter({ hasText: 'clear' }).click();
+    await page.getByRole('heading', { name: 'What does a data release look like?' }).click();
+    await page.getByText('Below are links out to datasets from other Count Me In projects. This is what we').click();
+    await page.getByRole('img', { name: 'The Metastatic Breast Cancer Project data release diagram' }).click();
+    await page.getByRole('img', { name: 'The Metastatic Prostate Cancer Project data release diagram' }).click();
+    await page.getByRole('img', { name: 'The Angiosarcoma Project data release diagram' }).click();
+  
+});
+
+test('Brain enroll kid on their behalf', async ( {page}) => {
+    await page.goto(BRAIN_BASE_URL!);
+    const userEmail = generateEmailAlias(BRAIN_USER_EMAIL);
+    testutils.fillSitePassword(page, SITE_PASSWORD)
+    
+    await page.waitForTimeout(1000);
+    await page.getByRole('banner').getByRole('link', { name: 'Count Me In' }).click();
+    await page.getByText('My child has been diagnosed with a brain tumor').click();
+    await page.getByRole('button', { name: 'Next' }).click();
+    await page.getByLabel('Enter age').click();
+    await page.getByLabel('Enter age').fill('10');
+    await page.locator('#mat-input-2').selectOption('US');
+    await page.locator('#mat-input-3').selectOption('GA');
+    await page.getByRole('button', { name: 'Submit' }).click();
+    await page.getByPlaceholder('yours@example.com').click();
+    await page.getByPlaceholder('yours@example.com').fill(userEmail);
+    await page.getByPlaceholder('yours@example.com').press('Tab');
+    await page.getByPlaceholder('your password').fill(BRAIN_USER_PASSWORD!);
+    await page.getByPlaceholder('your password').press('Enter');
+    await page.getByRole('heading', { name: 'Consent & Assent Form' }).click();
+    await page.getByText('Please read through the consent form text below and click Next when you are done').click();
```

Comment:
since these are text instead of buttons, should we use something like isVisible() or expect().toBeVisible() instead of click()?

Reply:
just saw the playwright generator plan below - resolving convo